pkg/helm/helm3/helm3.go

```diff
@@ -230,9 +239,9 @@ func (h *Helm3Client) ListReleasesNames() ([]string, error) {
 		return nil, fmt.Errorf("helm list failed: %v\n%s %s", err, stdout, stderr)
 	}
 
-	list := []struct {
+	var list []struct {
```

Comment:
can you explain this please?

Reply:
Initializing a list to nil is preferable to using var rather than the assignment operator.